zendesk_tickets/views.py

```diff
@@ -1,8 +1,17 @@
-from django.shortcuts import render
+from urllib.parse import urlparse
+
+from django.forms.forms import NON_FIELD_ERRORS
 from django.http import HttpResponseRedirect
+from django.shortcuts import resolve_url, render
+from django.template.response import TemplateResponse
+from django.utils.http import is_safe_url
+from django.utils.translation import ugettext as _
+from requests.exceptions import HTTPError
 
 from .forms import TicketForm
 
+RETURN_URL_FIELD = 'come_from'
```

Comment:
what do you think of using `'next'` so that it's similar to login forms, i.e. the next page you get taken to?